src/apps/routes/schemas/app.clj

```diff
@@ -464,7 +464,10 @@
          {(optional-key :search)
           (describe String
             "The pattern to match in an App's Name, Description, Integrator Name, or Tool Name.")
-
+          (optional-key :start_date)
+          (describe Date "The App's stat start date")
+          (optional-key :end_date)
+          (describe Date "The App's stat end date")
```

Comment:
I think it might be easier to read if there are blank lines between each key-value pair, since each key-value is already listed on separate lines.

Also, it might be clearer if the docs mention that these dates will filter the app's stats. Maybe something like "Filters out the App's stats before this `start_date`" and "Filters out the App's stats after this `end_date`"?